docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst

```diff
@@ -0,0 +1,151 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Intermediated Settlement
+########################
+
+This tutorial expands upon the principles discussed in our previous
+:doc:`Internal Settlement <internal>` tutorial, providing an in-depth exploration of intermediated
+settlement involving a multi-level account hierarchy across different custodians.
+
+Understanding Intermediated Settlement with Examples
+****************************************************
+
+This tutorial features two exemplary scripts, ``runWrappedTransfersSettlement`` and
+``runRouteProviderSettlement``, illustrating how to settle a batch with multiple instructions and
+custodians.
+
+Each script commences with ``runSetupIntermediatedSettlement``, initiating parties, establishing an
+instrument issued by the Central Bank, and setting up an account hierarchy rooted in the Central
+Bank. This hierarchy includes two custodian banks, Bank1 and Bank2, and their respective clients
+Alice, Bob, and Charlie. Holdings are created for Alice@Bank1, Bank1@CentralBank, and Charlie@Bank2:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Intermediated.daml
+  :language: daml
+  :start-after: -- WRAPPED_TRANSFERS_SETUP_BEGIN
+  :end-before: -- WRAPPED_TRANSFERS_SETUP_END
+
+The below diagram illustrates the setup, where edges represent accounts and stars (*) denote
+holdings::
+
+  --
```

Comment:
You can wrap it into a code sample, if probably shows better

Reply:
thanks, I wrapped it in a .. code-block:: none